.github/workflows/sql-test-workflow.yml

```diff
@@ -31,6 +31,16 @@ jobs:
     - name: Run tests
       id: tests
       run: |
+        # Spotless
+        ./gradlew :opensearch:spotlessCheck              || echo "* Spotless failed for opensearch/src" > report.log
+        ./gradlew :sql:spotlessCheck                     || echo "* Spotless failed for sql/src" >> report.log
+        ./gradlew :ppl:spotlessCheck                     || echo "* Spotless failed for ppl/src" >> report.log
+        ./gradlew :core:spotlessCheck                    || echo "* Spotless failed for core/test" >> report.log
```

Comment:
Is `spotlessCheck` runs for src or for test or for both?

Reply:
I need to add it to each of their `build.gradle` for it to work

Reply:
but it'll be for both